packages/api-client/src/views/Request/RequestSection/RequestParams.vue

```diff
@@ -164,7 +166,7 @@ watch(
       <div
         class="text-c-2 flex whitespace-nowrap opacity-0 group-hover/params:opacity-100 has-[:focus-visible]:opacity-100 request-meta-buttons">
         <ScalarTooltip
-          v-if="params.length > 1"
+          v-if="showTooltip"
```

Comment:
how does moving to a computed fix this?

Reply:
I was wondering that as well 🤔

Reply:
it doesn't but the missed `values` addition does 🫡

Reply:
Oh vales not value lol my mistake